app/config.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, runtime_environment):
         self.system_profile_topic = os.environ.get("KAFKA_TOPIC", "platform.system-profile")
         self.bootstrap_servers = os.environ.get("KAFKA_BOOTSTRAP_SERVERS", "localhost:29092")
         self.event_topic = os.environ.get("KAFKA_EVENT_TOPIC", "platform.inventory.events")
+        self.secondary_topic_enabled = True if os.environ.get("KAFKA_SECONDARY_TOPIC_ENABLED") == "true" else False
```

Comment:
`os.environ.get("KAFKA_SECONDARY_TOPIC_ENABLED") == "true"` is already a boolean expression. No need to wrap it in a conditional statement that returns boolean.

Also, it is a good practice for boolean env variables to be case insensitive i.e. no matter if set to "TRUE" or "true" it should still work as expected. See e.g. `PAYLOAD_TRACKER_ENABLED` for how it's done.


```suggestion
        self.secondary_topic_enabled = os.environ.get("KAFKA_SECONDARY_TOPIC_ENABLED", "false").lower() == "true"
```